apps/mark-scan/fai-100-controller/src/virtual_keyboard.rs

```diff
@@ -0,0 +1,44 @@
+use std::{thread, time::Duration};
```

Comment:
This implementation is identical to https://github.com/votingworks/vxsuite/blob/main/apps/mark-scan/accessible-controller/src/device.rs#L33-L67